app/v1/controllers/DeleteOtherExpensesController.scala

```diff
@@ -0,0 +1,72 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers
+
+import cats.data.EitherT
+import javax.inject.{Inject, Singleton}
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent, ControllerComponents}
+import utils.Logging
+import v1.controllers.requestParsers.DeleteOtherExpensesRequestParser
+import v1.models.errors._
+import v1.models.request.deleteOtherExpenses.DeleteOtherExpensesRawData
+import v1.services.{DeleteOtherExpensesService, EnrolmentsAuthService, MtdIdLookupService}
+import cats.instances.future._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class DeleteOtherExpensesController @Inject()(val authService: EnrolmentsAuthService,
+                                              val lookupService: MtdIdLookupService,
+                                              parser: DeleteOtherExpensesRequestParser,
+                                              service: DeleteOtherExpensesService,
+                                              cc: ControllerComponents)(implicit ec: ExecutionContext)
+  extends AuthorisedController(cc) with BaseController with Logging {
+
+  implicit val endpointLogContext: EndpointLogContext =
+    EndpointLogContext(controllerName = "DeleteOtherExpensesController", endpointName = "delete-expenses-other")
+
+  def handleRequest(nino: String, taxYear: String): Action[AnyContent] =
+    authorisedAction(nino).async { implicit request =>
+      val rawData = DeleteOtherExpensesRawData(nino, taxYear)
+      val result =
+        for {
+          parsedRequest <- EitherT.fromEither[Future](parser.parseRequest(rawData))
+          serviceResponse <- EitherT(service.deleteOtherExpenses(parsedRequest))
+        } yield {
+          logger.info(
+            s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] - " +
+              s"Success response received with CorrelationId: ${serviceResponse.correlationId}")
+          NoContent.withApiHeaders(serviceResponse.correlationId)
+
+        }
+      result.leftMap { errorWrapper =>
+        val correlationId = getCorrelationId(errorWrapper)
+        errorResult(errorWrapper).withApiHeaders(correlationId)
+      }.merge
+    }
+
+  private def errorResult(errorWrapper: ErrorWrapper) = {
+    (errorWrapper.error: @unchecked) match {
+      case NinoFormatError | BadRequestError | TaxYearFormatError | RuleTaxYearRangeInvalidError => BadRequest(Json.toJson(errorWrapper))
+      case DownstreamError => InternalServerError(Json.toJson(errorWrapper))
+      case NotFoundError => NotFound(Json.toJson(errorWrapper))
+      case UnauthorisedError => Forbidden(Json.toJson(errorWrapper))
```

Comment:
Controller doesn't need an unauthorised error as authorisation is covered in the AuthService